com/metacube/DSA/Assignment6/Graph.java

```diff
@@ -0,0 +1,290 @@
+package GET2018.com.metacube.DSA.Assignment6;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Stack;
+
+/**
+ * Copyright (c) 2018 Metacube.com. All rights reserved.
+ * This class defines functions for implementation of graph.
+ * @author Chirag Jain
+ *
+ */
+public class Graph implements IGraph
+{
+
+    private int numberOfVertices;
+    private LinkedList<EdgeAndVertices> listOfEdges[];
+    private EdgeAndVertices edge;
+    private List<EdgeAndVertices> edges;
+
+    public LinkedList<EdgeAndVertices>[] getListOfEdges() {
+        return listOfEdges;
+    }
+
+    // Constructor to initialize graph with number of vertices and list of edges
+    @SuppressWarnings("unchecked")
+    public Graph(int numberOfVertices) {
+        this.numberOfVertices = numberOfVertices;
+
+        listOfEdges = new LinkedList[numberOfVertices];
+
+        // Initializing the list of vertices
+        for (int i = 0; i < this.numberOfVertices; i++) {
+            listOfEdges[i] = new LinkedList<>();
+        }
```

Comment:
No loops, processing etc to be done in constructor. Create a separate method and call it here if needed.